src/cycax/cycad/features.py

```diff
@@ -145,13 +145,20 @@ class NutCutOut(Location):
 
     """
 
-    nuts = {
-        "type": 3,
-        "diameter": 6.01,
-        "height": 2.4,
+
+    nut_specifications={ #This is a global variable that will be used to cut the nuts by the OpenSCAD engine.
```

Comment:
Nice.
Probably use this in other Engines beside Openscad.
Is height the same as thickness. It is more common to talk about the thickness of a nut.